spec/ingreedy_spec.rb

```diff
@@ -229,6 +227,106 @@
   end
 end
 
+
+describe "with 'a' as quantity and preposition 'of'" do
+  before(:all) { @ingreedy = Ingreedy.parse "a dash of ginger" }
+
+  it "should have the correct amount" do
+    @ingreedy.amount.should == 1
+  end
+
+  it "should have the correct unit" do
+    @ingreedy.unit.should == :dash
+  end
+
+  it "should have the correct ingredient" do
+    @ingreedy.ingredient.should == "ginger"
```

Comment:
Lol, I'd like to say I did my research but that b/w avatar doesn't contain much information of that sort :laughing: